.gitignore

```diff
@@ -47,6 +47,7 @@
 !/lib/plugins/popularity
 !/lib/plugins/revert
 !/lib/plugins/safefnrecode
+!/lib/plugins/styler
```

Comment:
The name is in spirit with "config" and "extension". There was never a "original name". I started with styler which was taken.